partcad/src/partcad/test/cad.py

```
@@ -0,0 +1,25 @@
+# Licensed under Apache License, Version 2.0.
+#
+
+from .test import Test
+from .. import logging as pc_logging
+
+
+class CadTest(Test):
+    def __init__(self) -> None:
+        super().__init__("cad")
+
+    async def test(self, tests_to_run: list[Test], ctx, shape, test_ctx: dict = {}) -> bool:
```

Ruff (0.8.2)

Unused method argument: `tests_to_run`

(ARG002)